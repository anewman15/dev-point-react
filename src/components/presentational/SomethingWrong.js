import React from 'react';

const SomethingWrong = () => (
  <div>
    <h1>Something Went Wrong</h1>
    <p>Please try refreshing and make sure you&apos;re connected to the Internet</p>
  </div>
);

export default SomethingWrong;
