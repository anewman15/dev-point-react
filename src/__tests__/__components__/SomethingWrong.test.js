import { unmountComponentAtNode } from 'react-dom';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import SomethingWrong from '../../components/presentational/SomethingWrong';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('SomethingWrong', () => {
  it('renders the SomethingWrong component to the dom', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <SomethingWrong />
        </MemoryRouter>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
