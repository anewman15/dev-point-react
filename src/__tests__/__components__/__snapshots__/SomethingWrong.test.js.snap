// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomethingWrong renders the SomethingWrong component to the dom 1`] = `
<div>
  <h1>
    Something Went Wrong
  </h1>
  <p>
    Please try refreshing and make sure you're connected to the Internet
  </p>
</div>
`;
